### PR TITLE
Automate Grist deployment using ecosyste.ms metadata

### PR DESCRIPTION
**Pull request by Ly0n**
To keep the Grist spreadsheet up to date, and to show the additional ecosyste.ms data like download numbers, I wrote a Python script that reformats metadata from the ecosyste.ms API to fit into the spreadsheets. Current features:

1. The tables are not completely re-created, only the columns and rows are updated. This preserves all settings between deployments, such as sorting. All types must also be set manually via Grist. Setting column types via the API did not work so far. 
2. Manually created organisation labels like form_of_organisations are merged into the ecosyste.ms data. 
3. Cleaned up URLs so that they appear as links in Grist.  
4. We now have another column with the total number of projects listed per organisation. 
5. The CI is triggered once a week on Monday at 0:00 CET. 

Currently, the script is loaded into the CI job via wget from AwesomeCure, including the manually created labels. 
* Test deployment via Grist: https://docs.getgrist.com/8YWKLVW6EKD7/ostdeployment/p/2
* Successful CI: https://github.com/protontypes/open-sustainable-technology/actions/runs/10488736914/job/29053627575
* Python script: https://github.com/protontypes/AwesomeCure/blob/main/ecosystems_to_grist.py
* Handmade labels: https://github.com/protontypes/AwesomeCure/blob/main/csv/organizations_labeled.csv

I also added a note about the ecosyste.ms integration to our README and removed the CSV folder as this data is no longer needed.

### CODE REVIEW

File: .github/workflows/update_grist.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# Controls when the workflow will run
+#on:
+#  schedule:
+#    # Run at 00:00 UTC every Monday
+#    - cron: '0 0 * * 1'
+
+on:
+  push:
+    branches:
+      - ecosystems-to-spreadsheet-integration
```
Comment (andrew) [inline .github/workflows/update_grist.yml:11]: this will need to be updated to use the above cron schedule

**Review by andrew — CHANGES_REQUESTED** [review]

**Comment by andrew** [comment]
I wonder if it would make sense to move [ecosystems_to_grist.py](https://github.com/protontypes/AwesomeCure/blob/main/ecosystems_to_grist.py) into this repo?

**Comment by Ly0n** [comment]
> I wonder if it would make sense to move [ecosystems_to_grist.py](https://github.com/protontypes/AwesomeCure/blob/main/ecosystems_to_grist.py?rgh-link-date=2024-08-21T13%3A12%3A46Z) to this repo?

Right. We can add it next to the GitHub action yml to make it more visible what is working together. Should we also add the CSV file with the manually created labels to the yml? Then we have everything in one place that works together. 

I also notice 4 other things that need to be adjusted. 

- [ ] We need to change the document ID so that the old document is updated. 
- [ ] At the moment the category and subcategory for the organisations are a single label but some organisations work in multiple categories. I'm going to add the other categories as well.
- [ ] We also want to map organisation --> projects. I will create a column that just shows the git links related to an organisation. 
- [ ] We need the CSV file with the manually created to be recreated by the script so that new organizations are also included.

**Comment by Ly0n** [comment]
I added some more features: 

- [x] Merge the images into the CSV.
- [x] Add total_number_of_dependencies
- [x] Add package managers used by the project.

**Comment by andrew** [comment]
Makes it much harder to review when your code changes are in another repo btw

**Comment by Ly0n** [comment]
Sorry about that @andrew. I was using the AwesomeCure repo to test and hack various scripts related to metadata. I have now moved the script to this repo with the latest commit, including the CSV with the additional organisation labels we created in the past. The PR is now ready for review.

**Review by andrew — APPROVED** [review]

**Pull request merged**